### ISSUE TITLE
Issue on page /notebooks/blackmarblepy.html

### ISSUE THREAD

**Issue by Holly-Transport** [worldbank/blackmarblepy#100]
In the past, the daily and monthly illustrations showed gaps in the data (caused by clouds), which was really helpful for instructing others on the the pros and cons of different data frequencies. But the current images show these gaps filled in, which makes it harder to convey the idea of data gaps:
<img width="831" alt="image" src="https://github.com/user-attachments/assets/fd97daf6-69d5-4b6e-8902-21acfe0f8c21">

**Comment by g4brielvs** [comment]
> In the past, the daily and monthly illustrations showed gaps in the data (caused by clouds), which was really helpful for instructing others on the the pros and cons of different data frequencies. But the current images show these gaps filled in, which makes it harder to convey the idea of data gaps: 

[This page](https://worldbank.github.io/blackmarblepy/notebooks/blackmarblepy.html#create-raster-of-nighttime-lights) demonstrates the use of the package. For a more detailed discussion on data quality, including the impact of cloud cover and data gaps, please refer to [Assessing Quality of Nighttime Lights Data](https://worldbank.github.io/blackmarblepy/notebooks/quality-assessment.html#nighttime-lights).

**Comment by Holly-Transport** [comment]
Yes - thank you. It was very helpful when introducing the package to others to show, more clearly, what the gap-filling function is doing for daily and monthly pulls. If it's possible to add the not-filled daily, monthly, and annual images side-by-side, I think that would go far to prevent misuse of the package.
